qiita_ware/dispatchable.py

```diff
@@ -42,3 +42,170 @@ def create_raw_data(filetype, prep_template, filepaths):
 def copy_raw_data(prep_template, artifact_id):
     """Creates a new raw data by copying from artifact_id"""
     Artifact.copy(Artifact(artifact_id), prep_template)
+
+
```

Comment:
It seems like all these functions could also be benefitted from an approach like #1717